src/widget.tsx

```diff
@@ -119,11 +124,29 @@ export class CommentWidget<T> extends ReactWidget {
       );
     };
 
+    // This should be able to work without the bind?
```

Comment:
the `_CommentWrapper` function has lots of references to `this` inside, so it needs the bind.

Reply:
I see, when I was testing it myself I couldn't seem to find anything breaking without the bind.